src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/advanced-features.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Advanced features"
+metaDescription: "Manually record session replays, disable automatic session replay tracking and use nerd graph to configure replay settings."
+freshnessValidatedDate: 2024-12-19
+---
+
+## Manually record session replays [#manual-replays]
+
+If needed, you can programmatically `force` a session replay to start or stop recording using API methods. Here are some use cases:
+
+* Allow users to opt in for interaction recording. For example, if you have a pop-up asking "Allow this session to be recorded for performance analysis?" and a user consents:
+  1. Call `newrelic.recordReplay()` to initiate recording.
+  2. Once the session is complete or consent is withdrawn, use `newrelic.pauseReplay()` to stop capturing data.
+* Record sessions based on specific criteria, such as:
+  * Start recording only for sessions on specific URLs such as a product checkout flow.
+  * Stop recording for sensitive pages such as `/billing` or `/health-records`.
+
+See the API documentation for instructions:
```

Comment:
Global comment...use "refer" instead of "see"

Reply:
Updated